src/validation.h

```diff
@@ -161,8 +161,13 @@ struct MempoolAcceptResult {
         VALID, //!> Fully validated, valid.
         INVALID, //!> Invalid.
         MEMPOOL_ENTRY, //!> Valid, transaction was already in the mempool.
+        DIFFERENT_WITNESS, //!> Not validated. A same-txid-different-witness tx (see m_other_wtxid) already exists in the mempool and was not replaced.
     };
+    /** Result type. Present in all MempoolAcceptResults. */
     const ResultType m_result_type;
+
+    // The following field is only present when m_result_type = ResultType::INVALID
```

Comment:
field seems to exist in either case?

Reply:
Ah yeah, not nonexistent, but unset. Default is valid, with all fields empty, so not much meaning.

Reply:
Removed this comment